pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "libsvmdata"
+dynamic = ["version"]
```

Comment:
I think you also need to specify where it is defined, like we do here : https://github.com/scikit-learn-contrib/skglm/blob/main/pyproject.toml#L36

Reply:
yep, so far it's thinking this is version 0.0.0:
```
      Successfully uninstalled libsvmdata-0.5.dev0
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
celer 0.7.3.dev0 requires libsvmdata>=0.3, but you have libsvmdata 0.0.0 which is incompatible.
Successfully installed libsvmdata-0.0.0
```

Reply:
Of course, thanks! Fixed it now.